dffml/service/dev.py

```diff
@@ -908,4 +1041,5 @@ class Develop(CMD):
     setuppy = SetupPy
     bump = Bump
     ci = CI
+    lintcommits = LintCommits
```

Comment:
```suggestion
    lint = Lint
```

Let's make a `lint` command, then put `commits` under that.

We will use stuff like this in CI/CD for the plugins as we split them out into the 2ndparty org